src/j1cli.ts

```diff
@@ -1,15 +1,14 @@
-'use strict';
+import { JupiterOneClient } from './j1client';
+import path from 'path';
+import inquirer from 'inquirer';
+import * as error from './util/error';
+import fs from 'fs';
+import util from 'util';
+import yaml from 'js-yaml';
+import { defaultAlertSettings } from '@jupiterone/jupiterone-alert-rules';
+import pAll from 'p-all';
```

Comment:
It doesn't matter too much but I prefer the imports be sorted (third-party packages together, `@jupiterone/` packages together, and local imports together, with spaces in between). Just something to consider for the future (or disagree with me on).

Reply:
I'll do that rnrq

Reply:
K, grouped the deps.

Reply:
Oh, I didn't actually sort by dependency name. I wasn't sure if you preferred that or not when you said "sort". tbh doing that seems unnecessary to me

Reply:
No, not by name, that's too annoying to maintain. I just meant logical groups like what you did. Thanks!

Reply:
I use this: https://marketplace.visualstudio.com/items?itemName=mike-co.import-sorter

Reply:
Oh sweet. I'll have to look for a `vim`/`neovim` alternative

Reply:
> visualstudio.com

who let this guy into the building?